fiatconnect-api.md

```diff
@@ -919,9 +919,9 @@ the server MUST return a `FiatNotSupported` error.
 If the request is missing any required parameters, or if the parameters are poorly formed, the server MUST respond
 with an `InvalidParameters` error.
 
-#### 3.4.1.2. `GET /quote/out`
+#### 3.4.1.2. `POST /quote/out`
 
-The `GET /quote/out` endpoint is used to retrieve quotes used for transfers out from crypto to fiat currencies.
+The `POST /quote/out` endpoint is used to retrieve quotes used for transfers out from crypto to fiat currencies.
 
 ##### 3.4.1.2.1. Parameters
```

Comment:
the parameters below are listed as query parameters, but they should probably be given in the request body now that it's a post endpoint

Reply:
same for quote/in